README.md

```diff
@@ -77,14 +78,34 @@ api.list_farms()
 # {'farms': [{'farmId': 'farm-1234567890abcdefg', 'displayName': 'my-first-farm', ...},]}
 ```
 
-## Job attachments
+## Job-related files
+For job-related files and data, AWS Deadline Cloud supports either transferring files to AWS using job attachments or reading files from network storage that is shared between both your local workstation and your farm.  
 
-Job attachments enable you to transfer files between your workstations and AWS Deadline Cloud, by using Amazon S3 buckets as
+### Job attachments
+
+Job attachments enable you to transfer files between your workstations and AWS Deadline Cloud using Amazon S3 buckets as
 [content-addressed storage][cas] in your AWS account. The use of a content-addressed storage means that a file will never need
 to be uploaded again once it has been uploaded once.
 
 See [job attachments][job-attachments] for a more in-depth look at how files are uploaded, stored, and retrieved.
 
+### Shared storage and storage profiles
+Jobs can reference files that are stored on shared network storage. The Deadline Client uses a storage profile to determine which paths on the workstation are part of the network storage and do not need to be transferred using job attachments.
+
+To use an existing storage profile with the Deadline Client, you can configure your default storage profile via CLI:
+
+```sh
+deadline config set settings.storage_profile_id sp-10b2e48ad6ac4fc88595dfcbef6271f2
+```
+
+Or with the configuration GUI:
+```sh
+deadline config gui
+```
+
+
+To set up shared storage for your farm, you will need to set up a customer-managed fleet with shared network storage and to create a storage profile. See [shared storage](shared-storage) for more information.
```

Comment:
nit
```
To set up shared storage for your farm, you will need to set up a customer-managed fleet with shared network storage, and create a storage profile. See [shared storage](shared-storage) for detailed instructions.
```

Almost wonder if we should remove some information here because it doesn't include everything you need. Partial information might just add confusion. Maybe just:

```
For information on how to set up shared storage, see the [user guide].
```
I'm open to suggestions here.

Reply:
I wanted to add enough info so that most readers could quickly tell if storage profiles are necessary for their use case. If we just add a link to shared storage, someone that's not familiar with CMF vs SMF or job attachments vs shared storage will need to read a larger amount of documentation before learning if it's relevant.

What about something like, "Shared storage is possible with CMF but not SMF. For information on how to set up..." Or have any other ideas?